src/it/scala/io/iohk/ethereum/txExecTest/util/FixtureProvider.scala

```diff
@@ -62,19 +62,19 @@ object FixtureProvider {
 
       def traverse(nodeHash: ByteString): Unit = fixtures.stateMpt.get(nodeHash).orElse(fixtures.contractMpts.get(nodeHash)) match {
         case Some(m: BranchNode) =>
-          blockchain.nodesKeyValueStorageFor(Some(block.header.number), storages.nodeStorage, withSnapshotsSave = true).update(Nil, Seq(ByteString(m.hash) -> m.toBytes))
+          blockchain.nodesKeyValueStorageFor(Some(block.header.number), storages.nodeStorage).update(Nil, Seq(ByteString(m.hash) -> m.toBytes))
           m.children.collect { case Some(Left(hash)) => hash}.foreach(e => traverse(e))
 
         case Some(m: ExtensionNode) =>
-          blockchain.nodesKeyValueStorageFor(Some(block.header.number), storages.nodeStorage, withSnapshotsSave = true).update(Nil, Seq(ByteString(m.hash) -> m.toBytes))
+          blockchain.nodesKeyValueStorageFor(Some(block.header.number), storages.nodeStorage).update(Nil, Seq(ByteString(m.hash) -> m.toBytes))
           m.next match {
             case Left(hash) if hash.nonEmpty => traverse(hash)
             case _ =>
           }
 
         case Some(m: LeafNode) =>
           import AccountImplicits._
-          blockchain.nodesKeyValueStorageFor(Some(block.header.number), storages.nodeStorage, withSnapshotsSave = true).update(Nil, Seq(ByteString(m.hash) -> m.toBytes))
+          blockchain.nodesKeyValueStorageFor(Some(block.header.number), storages.nodeStorage).update(Nil, Seq(ByteString(m.hash) -> m.toBytes))
```

Comment:
maybe you could extract this to some `def` because this combination is used 3 times in here